core/src/main/java/com/exadel/etoolbox/backpack/core/servlets/model/PathModel.java

```diff
@@ -69,4 +72,10 @@ public boolean includeChildren() {
     public boolean includeLiveCopies() {
         return includeLiveCopies;
     }
+
+    /**
+     * Gets the whether or not references of the page included to the package build
+     * @return boolean value
+     */
+    public boolean isIncludeReferences() {return includeReferences;}
```

Comment:
(naming). Just `includeReferences` (see `includeLiveCopies()` above). "Is include" something is a grammar mistake)